### ISSUE TITLE
vagrant-vmware-desktop (3.0.2): Stdout: Invalid host type 'player' specified

### ISSUE THREAD

**Issue by EnorMOZ** [hashicorp/vagrant-vmware-desktop#84]
### Vagrant version

```
❯ vagrant -v
Vagrant 2.3.4
```

### Vagrant VMware plugin version

```
❯ vagrant plugin list
vagrant-vmware-desktop (3.0.2, global)
  - Version Constraint: > 0
 ```

### Vagrant VMware utility version

```
❯ /opt/vagrant-vmware-desktop/bin/vagrant-vmware-utility -v
1.0.21
```

### Fusion version

```
Player Version 13.0.2
```

### Host operating system

```
MacOs 13.3.1
```

### Guest operating system

Any

### Vagrantfile

```ruby
Vagrant.configure("2") do |config|
    config.vm.box = "starboard/ubuntu-arm64-20.04.5"
    config.vm.box_version = "20221120.20.40.0"
    config.vm.network "private_network", ip: "10.10.10.100", auto_config: false

    config.vm.provider "vmware_desktop" do |v|
      v.gui = true
      v.vmx["ethernet0.virtualdev"] = "vmxnet3"
      v.vmx["ethernet0.pcislotnumber"] = "160"
    end
  end
```

### Debug output

[vagrant.log](https://github.com/hashicorp/vagrant-vmware-desktop/files/11424326/vagrant.log)

### Expected behavior

Starts up

### Actual behavior
```
❯ vagrant up
Bringing machine 'default' up with 'vmware_desktop' provider...
==> default: Cloning VMware VM: 'starboard/ubuntu-arm64-20.04.5'. This can take some time...
An error occurred while executing `vmrun`, a utility for controlling
VMware machines. The command and output are below:

Command: ["-T", "player", "snapshot", "/Users/userA/.vagrant.d/boxes/starboard-VAGRANTSLASH-ubuntu-arm64-20.04.5/20221120.20.40.0/vmware_fusion/ubuntu-arm64-20.04.5-3.vmx", #<Pathname:9a2f0eaf-4701-44ad-bddc-7242883db0fb>, {:notify=>[:stdout, :stderr]}]

Stdout: Invalid host type 'player' specified
Error: The operation is not supported

Stderr:
```
### Steps to reproduce

1. Use vagrant-vmware-desktop (3.0.2)
2. vagrant up
3. Revert back to vagrant-vmware-desktop (3.0.1), works

**Comment by EnorMOZ** [comment]
```
vmrun version 1.17.0 build-21581413

Usage: vmrun [AUTHENTICATION-FLAGS] COMMAND [PARAMETERS]



AUTHENTICATION-FLAGS
--------------------
These must appear before the command and any command parameters.

   -T <hostType> (ws|fusion)
```
Looks like player is not available ?

**Comment by EnorMOZ** [comment]
Tried 
```
❯ /opt/vagrant-vmware-desktop/bin/vagrant-vmware-utility -v
1.0.22
```
Same results with it failing.  Reverting the plugin back to 3.0.1 allows it to work again.

**Comment by sbailliez** [comment]
Can confirm I had the same issue when upgrading vagrant-vmware-desktop to 3.0.2 on macOS 13.3.1 (a) and it worked after reverting to 3.0.1

**Comment by chrisroberts** [comment]
Hi there,

This is a duplicate of #78 and was fixed with #79. It was included in version 3.0.3 of the vagrant-vmware-desktop, so updating the plugin should resolve your issue.

Cheers!
